src/safeposix/cage.rs

```diff
@@ -49,7 +49,8 @@ pub struct PipeDesc {
     pub flags: i32
 }
 
-type FdTable = interface::RustHashMap<i32, interface::RustRfc<interface::RustLock<FileDescriptor>>>;
+pub type FdTable = interface::RustHashMap<i32, interface::RustRfc<interface::RustLock<FileDescriptor>>>;
```

Comment:
why does this need to be pub?

Reply:
I think that was an accident on my part -- probably was matching wrong